packages/notifications/src/inAppMessaging/InAppMessaging.ts

```diff
@@ -57,15 +55,10 @@ class InAppMessagingClass {
 		listenForAnalyticsEvents = true,
 		onMessagesReceived,
 		...config
-	}: InAppMessagingConfig = {}) => {
-		this.config = Object.assign(
-			{},
-			this.config,
-			parseMobileHubConfig(config).Notifications?.InAppMessaging ?? {},
-			config
-		);
+	}: InAppMessagingConfig = {}): InAppMessagingConfig => {
+		this.config = Object.assign({}, this.config, config);
```

Comment:
Can we replace the usage of `Object.assign` with a spread operator? (non blocking)

Reply:
I don't have a preference in these cases, I can make the change